projects/common/src/utilities/formatters/time/display-time-ago.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Dictionary } from '../../types/types';
+
+@Pipe({
+  name: 'htDisplayTimeAgo'
+})
+export class DisplayTimeAgo implements PipeTransform {
+  private readonly intervals: Dictionary<number> = {
+    year: 31536000,
```

Comment:
This stuff is already built into duration, can't we just reuse that (`TimeDuration.getMostSignificantUnitOnly`)